pyxnat/core/pipelines.py

```diff
@@ -2,14 +2,20 @@
 import os
 import json
 import datetime
-import urllib2
-import smtplib
-from copy import deepcopy
-import email.mime.text
-import xml.dom.minidom
 
-import suds.client
-import suds.xsd.doctor
+import six
+if six.PY2:
```

Comment:
I don't think `urlopen` is ever used

Reply:
Actually it is used further down in the commented part. The previous import urlopen was failing due to Python 3 incompatibility. Shall we just comment the whole import?

Reply:
Agree